File: practias_logica_programacion/javaScript_practicas/ejercicio2_palindromo.js
'use strict'

function palindromo(texto){

    let textArray = texto.split("");
    let array_reverse = textArray.reverse();
    let array_string =array_reverse.join("")

    console.log(texto)
    console.log(array_string)

    if(texto == array_string){
        console.log('es un palindromo');
    }else{
        console.log('no es un palindromo');
    }
    
    
}

palindromo('reconocer');

File: practias_logica_programacion/python_practicas/ejercicio2_palindromo.py
def palindromo(texto):

    text_list = list(texto)

    inver_text_list = ''.join(text_list[::-1])

    if texto == inver_text_list:
        print('Es un palindromo')

    else:
        print('no es un palindromo')

    #print(inver_text_list)
palindromo('reconocer')

